### PR TITLE
add a landing page

### PR DESCRIPTION
**Pull request by flynnduism**
This PR adds a simple landing page (via github pages) with some high level info about the project.

# preview

Screen Shot 2019-05-15 at 1 45 49 PM

The landing page can be viewed on [github pages here](https://flynnduism.github.io/vscode-kubernetes-tools/) (my fork)

https://flynnduism.github.io/vscode-kubernetes-tools/

# follow up:

- [x] review the copy and layout
- [ ] get a better screenshot w/deeper cluster data
- [x] update the [baseURL](https://github.com/flynnduism/vscode-kubernetes-tools/blob/landing-page/site/config.toml#L1) to use the azure fork

### CODE REVIEW

File: site/themes/vscode/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+![Theme Screenshot](https://raw.githubusercontent.com/deislabs/cnab.io/master/themes/cnab/img/screenshot.png?token=AAp4ctyXT6Xuaer1Vm8SXT0HscNhS5WGks5cBvT9wA%3D%3D)
+
+CNAB Theme
```
Comment (itowlson) [inline site/themes/vscode/README.md]: Should we use a different name here, or is this the name of the theme?  I'm guessing it's not very visible so won't cause confusion.

File: site/themes/vscode/layouts/index.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    <div class="row">
+      <div class="show-for-large-up large-1 columns">&nbsp;</div>
+      <section class="small-12 large-5 columns">
+        <h1>
+          <span class="strike">Infrastructure as code</span><br>
+          Infrastructure in <a href="https://twitter.com/code" target="_blank">@code</a>
+        </h1>
+        <!-- <h2>Access key parts of the Kubernetes ecosystem right inside your code editor.</h2> -->
+        
+        <p>The <a href="https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools">Kubernetes Tools</a> extension for VS Code bundles together a whole ecosystem of useful open source tools, making it easy for you to access and manage key parts of your app and microservices without leaving your code.</p>
```
Comment (itowlson) [inline site/themes/vscode/layouts/index.html]: I'm not sure we 'bundle a whole ecosystem' - longer term we will have packs that bundle larger parts of the ecosystem but the core is only Helm, and Draft.  @squillace may have thoughts on the wording here.

**Comment by flynnduism** [comment]
Thanks @itowlson. Changes made:

* added `site` to `.vscodeignore`
* reworded that bit around _"ecosystem"_ to make it simpler. awaiting ralpha judgements
* fixed the theme description

**Comment by itowlson** [comment]
@flynnduism @squillace Is this ready to merge, so we can have it live in time for the Kubecon talk?

**Comment by flynnduism** [comment]
@itowlson happy to proceed if @squillace is satisfied

File: site/themes/vscode/layouts/index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        <p>In building the kubernetes-tools extension, we've extracted a helpful <a href="https://github.com/Azure/vscode-kubernetes-tools-api">kubernetes-tools-api</a> layer and published it <a href="https://github.com/Azure/vscode-kubernetes-tools-api">here</a>, with the aim of enabling the development of more custom Kubernetes tools.</p>
+
+        <p>The API is available as an <a href="https://www.npmjs.com/package/vscode-kubernetes-tools-api">NPM package</a>, so you can easily import and build upon it in your own extensions!</p>
+
+        <code>npm -i vscode-kubernetes-tools-api</code>
+
+        <h2>Additional info:</h2>
+        <ul>
+          <li>
+            <a href="https://github.com/Azure/vscode-kubernetes-tools/tree/master/docs">Documentation</a>
```
Comment (itowlson) [inline site/themes/vscode/layouts/index.html]: Can we change this to https://github.com/Azure/vscode-kubernetes-tools/tree/master/docs/extending please?

File: site/themes/vscode/layouts/index.html [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+              <a href="https://github.com/Azure/vscode-kubernetes-tools">Github: vscode-kubernetes-tools</a>
+          </li>
+          </li>
+          <li>
+            <a href="https://github.com/Azure/vscode-kubernetes-tools-api">Github: vscode-kubernetes-tools-api</a>
+          </li>
+          <li>
+            <a href="https://github.com/Azure/vscode-kubernetes-tools-api">NPM Package</a>
+          </li>
+        </ul>
```
Comment (itowlson) [inline site/themes/vscode/layouts/index.html]: Could we also add a link to the API samples please?  https://github.com/Azure/vscode-kubernetes-tools-api-samples

File: site/themes/vscode/layouts/index.html [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+<div class="row row-full full-height" id="api">
+  <section class="small-12 columns">
+
+    <div class="row">
+      <div class="show-for-large-up large-1 columns">&nbsp;</div>
+      <section class="small-12 large-6 columns">
+        <h1>Kubernetes Tools API</h1>
+        <h2>Custom tooling on top of the Kubernetes Tools extension.</h2>
+
+        <p>In building the kubernetes-tools extension, we've extracted a helpful <a href="https://github.com/Azure/vscode-kubernetes-tools-api">kubernetes-tools-api</a> layer and published it <a href="https://github.com/Azure/vscode-kubernetes-tools-api">here</a>, with the aim of enabling the development of more custom Kubernetes tools.</p>
```
Comment (itowlson) [inline site/themes/vscode/layouts/index.html]: The two links here are probably redundant - you could change the 'published it here' link to point to the NPM package?

**Comment by flynnduism** [comment]
thanks for the comments @itowlson - I've made those url changes as suggested!

**Comment by itowlson** [comment]
Thanks for the merge - sorry for dropping the ball on this.

**Pull request merged**